### PR TITLE
Remove warnings in case of error `tfw_http_req_cache_cb`

### PR DESCRIPTION
**Pull request by EvgeniiMekhanik**
There are to may warnings in dmesg if there is no appropriate backend server, so print only debug message.

### CODE REVIEW

File: fw/http.c [hunk 1]
```diff
@@ -5497,11 +5497,13 @@ tfw_http_req_cache_cb(TfwHttpMsg *msg)
...
 
 send_502:
-	tfw_http_send_err_resp(req, 502, "request dropped: processing error");
+	T_DBG("request dropped: processing error, status 502");
+	tfw_http_send_err_resp(req, 502, NULL);
 	TFW_INC_STAT_BH(clnt.msgs_otherr);
 	return;
 send_500:
-	tfw_http_send_err_resp(req, 500, "request dropped: processing error");
+	T_DBG("request dropped: processing error, status 500");
```
Comment (krizhanovsky) [inline fw/http.c:5505]: I'm OK with this version, we already have a `tfw_http_send_err_resp()` with NULL `reason`. However, I'd prefer a way proposed by @const-t to introduce a function like `tfw_http_send_err_resp_nolog()`, which don't accept `reason` and which is called by `tfw_http_send_err_resp()` for the actual part of the sending

**Review by krizhanovsky — APPROVED** [review]
LGTM

File: fw/http.c (hunk 1 above)
Comment (EvgeniiMekhanik) [reply fw/http.c:5511]: Yes fixed

**Review by const-t — APPROVED** [review]
LGTM

**Pull request merged**